renormalizer/model/model.py

```diff
@@ -399,11 +399,11 @@ class TI1DModel(Model):
     The Hamiltonian should take the form:
 
     .. math::
-        \hat H = \sum_i(\hat h_i + \sum_j \hat h_{i, i+j})
+        \hat H = \sum_i(\hat h_i + \sum_j \hat h_{i,j})
 
     where :math:`\hat h_i` is the local Hamiltonian acting on one single unit cell
-    and :math:`\hat h_{i, i+j}` represents the interaction between the :math:`i` th cell
-    and the :math:`i+j` th cell.
+    and :math:`\hat h_{i,j}` represents the :math:`j` th interaction between the :math:`i` th cell
+    and other unit cells.
```

Comment:
The doc here confuses me. 
I think `j` does not represent the `j` th interaction. Instead, it is the index of  `j`th site, representing the two-site interaction between `i`th and `j`th sites.

Reply:
The model is not limited to two-unitcell interaction and `h_{ij}` appears to be the most general way to express it.

Reply:
Got it!